src/monster.h

```diff
@@ -575,6 +575,7 @@ class monster : public Creature
         void on_load();
 
         const pathfinding_settings &get_pathfinding_settings() const override;
+        std::vector<std::string> get_trap_avoids() const;
```

Comment:
This makes a copy, that's expensive and unnecessary.
```suggestion
        const std::set<trap_id> &get_trap_avoids() const;
```

Reply:
Resolved